esp/src/src-react/components/Workunits.tsx

```diff
@@ -75,6 +79,7 @@ export const Workunits: React.FunctionComponent<WorkunitsProps> = ({
     const hasFilter = React.useMemo(() => Object.keys(filter).length > 0, [filter]);
 
     const [showFilter, setShowFilter] = React.useState(false);
+    const sashaService = React.useMemo(() => new SashaService({ baseUrl: "" }), []);
```

Comment:
Since this service never changes, it should be moved outside of the component. I'm pretty sure that's the pattern we use everywhere, or should be. So instead of the useMemo here, move `const sashaService = new SashaService({ baseUrl: "");`, somewhere up around `const logger = scoped...` And then sashaService can be removed from any dependency arrays later on